bigquery/indexer.py

```diff
@@ -82,17 +81,14 @@ def _get_nested_mappings(schema, prefix=None):
     return nested if nested else None
 
 
-def _create_nested_mappings(es, index_name, table_name, sample_id_col,
-                            billing_project_id):
+def _create_nested_mappings(es, index_name, table, sample_id_col):
     # Create nested mappings for repeated record type BigQuery fields so that
     # queries will work correctly, see:
     # https://www.elastic.co/guide/en/elasticsearch/reference/6.4/nested.html#_how_arrays_of_objects_are_flattened
-    project_id, dataset, table = table_name.split('.')
-    bq = bigquery.Client(project=billing_project_id)
-    t = bq.get_table(bq.dataset(dataset, project=project_id).table(table))
-    nested = _get_nested_mappings(t.schema, table_name)
+    table_name = _get_table_name(table.full_table_id)
```

Comment:
nit move to before comment on line 85.

Looks like this is used once, you could just inline
```
nested = _get_nested_mappings(table.schema, _get_table_name(table.full_table_id))
```